tests/Config/Parser/fixtures/annotations/Type/Lightsaber.php

```diff
@@ -19,20 +19,23 @@ class Lightsaber
      * @GQL\Field
      */
     #[GQL\Field]
+    // @phpstan-ignore-next-line
     protected $color;
 
     /**
      * @ORM\Column(type="text")
      * @GQL\Field
      */
     #[GQL\Field]
+    // @phpstan-ignore-next-line
     protected $text;
```

Comment:
```suggestion
    /**
     * @ORM\Column(type="text")
     * @GQL\Field
     * @phpstan-ignore-next-line
     */
    #[GQL\Field]
    protected $text;
```

Should also work

Reply:
I tried but it doesn't work. It seems to be because of the attribute line.